jujugui/static/gui/src/app/components/usso-login-link/usso-login-link.js

```diff
@@ -49,26 +49,43 @@ YUI.add('usso-login-link', function() {
     },
 
     /**
+       Returns a notification about popups
+     */
+    _renderNotification: function() {
+      return (
+        <div className="usso-login__notification">
+          When requested please allow popups from {window.location.origin} to login
+        </div>
+      );
+    },
+
+    /**
       Returns the text login link.
     */
     _renderTextLink: function() {
       return (
-        <a className={'logout-link'}
-          onClick={this._handleLogin}
-          target="_blank">
-          Login
-        </a>);
+        <div className="usso-login">
+          <a className={'logout-link'}
+            onClick={this._handleLogin}
+            target="_blank">
+            Login
+          </a>
+          {this._renderNotification()}
+        </div>);
     },
 
     /**
       Returns the button login link.
     */
     _renderButtonLink: function() {
       return (
-        <juju.components.GenericButton
-          action={this._handleLogin}
-          type="positive"
-          title="Sign up or Login" /> );
+        <div className="usso-login">
```

Comment:
This wrapper can move to the actual render methods instead of each one of the type renderings.